cwms-radar-client/src/main/java/mil/army/usace/hec/cwms/radar/client/controllers/LocationGroupEndpointInput.java

```diff
@@ -81,6 +92,8 @@ protected HttpRequestBuilder addInputParameters(HttpRequestBuilder httpRequestBu
             return httpRequestBuilder.addQueryParameter(GROUP_ID_QUERY_PARAMETER, groupId)
                     .addQueryParameter(OFFICE_QUERY_PARAMETER, officeId)
                     .addQueryParameter(CATEGORY_ID_QUERY_PARAMETER, categoryId)
+                    .addQueryParameter(GROUP_OFFICE_ID_QUERY_PARAMETER, groupOfficeId)
+                    .addQueryParameter(CATEGORY_OFFICE_ID_QUERY_PARAMETER, categoryOfficeId)
```

Comment:
Missing the `"include-assigned"`

Reply:
Only getAll has include-assigned as an input parameter